PickMeUp/PickMeUp/viewControllers/DM Features/PMConversationViewController.m

```diff
@@ -39,17 +50,47 @@ - (void)viewDidLoad {
     self.tableView.rowHeight = UITableViewAutomaticDimension;
     self.messageToSend.delegate = self;
     
+    if (self.convo != nil) {
+        [self _finishCreatingLiveQuery];
+    }
+    
+    [self _runGetQuery];
+    // creates bottom refresh control
+    UIRefreshControl *refreshControl = [UIRefreshControl new];
+    refreshControl.triggerVerticalOffset = 100.;
+    [refreshControl addTarget:self action:@selector(_runGetQuery) forControlEvents:UIControlEventValueChanged];
+    self.tableView.bottomRefreshControl = refreshControl;}
+
+-(void)_createPopUp:(NSString *)title message:(NSString *)message {
```

Comment:
nit: `create` prefix is normally used for functions that create and return instance. Let's rename it to something `show` or `presentPopup`